src/sagemaker/lineage/query.py

```diff
@@ -104,6 +104,18 @@ def __str__(self):
         """
         return str(self.__dict__)
 
+    def __repr__(self):
+        """Define string representation of ``Edge``.
+
+        Format:
+            {
+                'source_arn': 'string', 'destination_arn': 'string',
+                'association_type': 'string'
+            }
+
+        """
+        return "\n\t" + str(self.__dict__)
```

Comment:
__ repr__ of Vertices and Edges will automatically be called when LineageQueryResult str is called (LineageQueryResult has lists of Vertices and Edges), so thats why \n\t is added in repr so that when LineageQueryResult str is called, it won't be whole chuck of text instead it would contain indentation for Vertices and Edges and will be easier to read.